scrapers_next/tx/committees.py

```diff
@@ -0,0 +1,103 @@
+import re
+from spatula import HtmlPage, HtmlListPage, CSS, XPath, SelectorError
+from openstates.models import ScrapeCommittee
+
+
+class CommitteeDetail(HtmlPage):
+    example_source = "https://capitol.texas.gov/Committees/MembershipCmte.aspx?LegSess=87R&CmteCode=C014"
+
+    def process_page(self):
+        com = self.input
+        com.add_source(self.source.url)
+
+        table = XPath("//div[@id='content']/table[2]/tr[position()>1]").match(self.root)
+        for person in table:
+            name = person.text_content()
+
+            positions = ["chair", "vice chair"]
+            if name:
+                try:
+                    label_text = XPath("./td[1]//text()").match(person)
+                    for label in label_text:
+                        label = label.lower()
+                        if label.endswith(":"):
+                            label = label[:-1]
+                        if label in positions:
+                            role = label
+                        else:
```

Comment:
I'd maybe use a `self.warn(f"unknown role {label}")` or similar here so we can know what roles are skipped by looking at the output

Reply:
adjusted it a bit to only raise the warning when the label is not "members", and to set the role to "members" after sending the warning (just to be extra safe!)